apps/sms/js/information.js

```diff
@@ -146,29 +146,34 @@ function showSimInfo(element, iccId) {
     return;
   }
 
-  var info = [];
+  var simInfoElement = element.querySelector('.sim-detail');
   var simId = Settings.getServiceIdByIccId(iccId);
-  var operator = Settings.getOperatorByIccId(iccId);
-  var number = iccManager.getIccById(iccId).iccInfo.msisdn;
-  var data = {};
-  var l10nId;
 
-  info = [operator, number].filter(function(value) {
-    return value;
-  });
+  if (simId === null) {
+    navigator.mozL10n.setAttributes(
+      simInfoElement,
+      'dsds-unknown-sim'
```

Comment:
nit: I understand this only happend while dsds case that we need to show the sim info, but maybe we can simply use `unknown-sim` as id in general?